src/compas_rhino/install_with_pip.py

```diff
@@ -1,20 +1,117 @@
+import pathlib
+import random
+import string
 import subprocess
 
-import compas_rhino
+import click
```

Comment:
But click should be a regular requirement not a dev one, this install method is not a dev-only thing. Could we do it without taking a dependency on click?